src/ui/src/data/matchQueue.ts

```diff
@@ -1,35 +1,21 @@
-import { Record as RLRecord } from "@/models/record";
+import { RecordMatch } from "@/models/recordMatch";
```

Comment:
```suggestion
"use server";

import { RecordMatch } from "@/models/recordMatch";
```

Can we make this server side to allow for easier configuration and have the page render faster?

Reply:
We can't as the nextJS application is being build and exported as static assets and does not run on a nextJS server.

Reply:
Ah, right! You said that last week. Now I am seeing the ramifications of different set ups between dev and what will be deployed.